packages/loading/lib/loading.js

```diff
@@ -40,6 +46,38 @@ class LoadingUI {
     const indexPath = resolve(distPath, 'index.html')
     this.indexTemplate = await fs.readFile(indexPath, 'utf-8')
     this.app.use('/', this.serveIndex)
+
+    // Start listening
+    await this._listen()
+  }
+
+  _listen () {
+    return new Promise((resolve, reject) => {
+      if (this._server) {
+        return resolve()
+      }
+      this._server = this.app.listen(0, (err) => {
```

Comment:
We need a way to listen on the same port when the Nuxt server restart when updating the `nuxt.config.js`